docs/source/conf.py

```diff
@@ -80,7 +85,16 @@
     else:
         main_rst = os.path.join(output_dir,
                                 "avocado.%s.rst" % second_level_module_name)
-    main_rst_content = open(main_rst).readlines()
+    if not apidoc:
+        main_rst_content = []
+        try:
+            os.makedirs(os.path.dirname(main_rst))
+        except OSError as details:
+            if not details.errno == 17:
```

Comment:
Absolutely! Thanks for spotting this lame literal use (I actually hate code like this).